src/test/scripts/functions/external/FunInExpressions2.dml

```diff
@@ -0,0 +1,37 @@
+#-------------------------------------------------------------
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+# 
+#   http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+#-------------------------------------------------------------
+
```

Comment:
@dusenberrymw , can you please see the function check and advice on what all the checks need to be happened?

Reply:
What do you mean by "what all the checks need to happened" - having two test separate tests if fine. As a third one, you might want to include a recursive function just as a basis for future extensions because these require special treatment especially during IPA (i.e., they require a function stack to avoid cycles).